message.cpp

```diff
@@ -1,3 +1,8 @@
+/**
+ * Some of the content of this file has been edited by Metaswitch, in the time
+ * period from December 2012 to the present time.
+*/
+
```

Comment:
Would it make sense for this to be part of the larger block comment below?

Reply:
I intend to leave this, so that all files have a consistent message at the top

Reply:
Ok, makes sense 😺